src/srs-filter.c

```diff
@@ -670,6 +738,15 @@ void daemonize() {
     exit(EXIT_SUCCESS);
   }
 
+  /* update pidfile */
+  if (CONFIG_pidfile) {
+    FILE *f;
+
+    f = fopen(CONFIG_pidfile, "w");
```

Comment:
check for null and syslog an error if fopen fails -- continue running regardless, though?